vtkVmtk/ComputationalGeometry/vtkvmtkSimplifyVoronoiDiagram.cxx

```diff
@@ -182,7 +187,18 @@ int vtkvmtkSimplifyVoronoiDiagram::RequestData(
   currentPolys->DeepCopy(inputPolys);
 
   currentLinks->Allocate(input->GetNumberOfPoints());
+#if VTK_MAJOR_VERSION >= 9 || (VTK_MAJOR_VERSION >= 8 && VTK_MINOR_VERSION >= 90)
+  // Reworking cell links classes for performance and consistent API
+  // See https://github.com/Kitware/VTK/commit/88efc809a25130c2ab83dd89a80cea458e3bb56a
+  #pragma message "vtkvmtkSimplifyVoronoiDiagram::RequestData not functional. Must be updated based on Kitware/VTK@88efc809a"
+  if (true)
+    {
+    vtkErrorMacro(<< "vtkvmtkSimplifyVoronoiDiagram::RequestData is not functionnal when built against VTK >= 9");
+    return 0;
+    }
+#else
   currentLinks->BuildLinks(input,currentPolys);
+#endif
```

Comment:
I am not tracking what this warning message is saying?
that RequestData needs to be re-written to comply with upstream vtk changes?

If BuildLinks doesn't exist anymore in VTK9 will then calls to RequestData fail?

Reply:
Yes, unfortunately Voronoi smoothing currently is not available with VTK9. It would be great if you could fix it.

Reply:
so basically any reference to it is also broken. Got it.

Reply:
Do you have a reference for the implementation that might get me started on trying to fix it.

Reply:
Unfortunately, I don't know how to fix this. The API has changed fundamentally, so I think the best is to ask on the [VTK forum](https://discourse.vtk.org/). Please post the link here to the question that you ask there so that we can follow the discussion.

Reply:
I believe I was able to resolve this issue.